tests/unit/multiclass/BaggingMachine_unittest.cc

```diff
@@ -233,8 +263,96 @@ TEST(BaggingMachine,classify_CART)
 	CMulticlassAccuracy* eval=new CMulticlassAccuracy();
 	EXPECT_NEAR(0.642857,c->get_oob_error(eval),1e-6);
 
-	SG_UNREF(test_feats);
 	SG_UNREF(result);
 	SG_UNREF(c);
 	SG_UNREF(eval);
 }
+
+TEST_F(BaggingMachine, output_binary)
+{
+	sg_rand->set_seed(1);
+
+	SGVector<bool> ft=SGVector<bool>(4);
```

Comment:
can't this first bit also be put into a common place. e.g. hierarchy of fixtures? so it is not replicated below?